docs/developer/tutorials/build-rocks-in-launchpad.rst

```diff
@@ -0,0 +1,185 @@
+
+    # The web link to view the build request
+    >>> build_request.web_link
+    'https://launchpad.net/~username/project_name/+rock/test-rock/+build-request/id'
+
+You will need to query Launchpad to obtain the status of your build request
+(Pending, Failed, Completed), you will not be notified in the CLI once the
+build is completed::
+
+    >>> build_request.status
+    'Completed'
+
+You can try refreshing the object state at any time by running::
+    
+    >>> build_request.lp_refresh()
+
+To learn more about what can be done with the build object, refer to the
+`rock recipe build request API documentation <https://api.launchpad.net/devel.html#rock_recipe_build_request>`_.
+
+To get the builds produced by the build request::
+
+    >>> rock_build = build_request.builds
+
+    >>> rock_build
+    <lazr.restfulclient.resource.Collection at 0x...>
+
+Here, ``rock_build`` is a collection of builds based on the distribution series
+and architecture set.
+
+Again, you will need to query Launchpad to obtain the status of your build, you
+will not be notified in the CLI once the build is completed:
+
+    >>> for build in rock_build:
+    ...     build.lp_refresh()
+    ...     print(build.web_link, build.buildstate)
+    https://launchpad.net/~username/project_name/+rock/test-rock/+build/id Successfully built
+
+To learn more about what can be done with the build object, refer to the
+`rock recipe build API documentation
+<https://api.launchpad.net/devel.html#rock_recipe_build>`_.
+
+Download the Rock
+-----------------
+
+Once the build has completed successfully, the build artifacts can be obtained
+by::
+
+    for build in rock_build:
+    ...     print(build.getFileUrls(), build.build_log_url)
+    ['https://launchpad.net/~username/project_name/+rock/test-rock/+build/id/+files/hello_latest_arm64.rock']
+    https://launchpad.net/~username/project_name/+rock/test-rock/+build/id/+files/buildlog_rock_ubuntu_..._test-rock-1_BUILDING.txt.gz
+
+
+To download the rock, you can use `urllib.request
+<https://docs.python.org/3/library/urllib.request.html#module-urllib.request>`_::
+
+    >>> for build in rock_build:
+    ...   for url in build.getFileUrls():
+    ...     filename = url.split("/")[-1]
+    ...     urllib.request.urlretrieve(url, filename)
+    ...     print(f"Downloaded {filename}")
+    ... 
+    Downloaded hello_latest_arm64.rock
+
+Build Failures
+--------------
+
+In the case a build fails, ensure that the rock can be built locally by running
+the ``rockcraft pack`` command. You can go through the buildlog
+(``build.build_log_url``) and retry the build::
+
+    >>> for build in rock_build:
+    ...     build.retry()
+
+Next Steps
+----------
+
+- `Run your rock by following these steps
+  <https://documentation.ubuntu.com/rockcraft/stable/tutorial/hello-world/>`_
+- Learn how to build snaps and charms on Launchpad
```

Comment:
This needs to be a link/links, but probably cannot be one yet as the other docs are not yet released. IMHO we should remove this bullet point then until the other docs are available.